internal/pkg/aws/ec2/ec2.go

```diff
@@ -183,6 +187,33 @@ func (c *EC2) ListVPCs() ([]VPC, error) {
 	return vpcs, nil
 }
 
+// ListAZs returns the list of opted-in and available availability zones.
+func (c *EC2) ListAZs() ([]AZ, error) {
+	resp, err := c.client.DescribeAvailabilityZones(&ec2.DescribeAvailabilityZonesInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("zone-type"),
+				Values: aws.StringSlice([]string{"availability-zone"}),
+			},
+			{
+				Name:   aws.String("state"),
+				Values: aws.StringSlice([]string{"available"}),
+			},
```

Comment:
Would we need the "opt-in-status" filter?

Reply:
My understanding from the SDK description is that we don't need to: https://docs.aws.amazon.com/sdk-for-go/api/service/ec2/#DescribeAvailabilityZonesInput
```
    // Include all Availability Zones, Local Zones, and Wavelength Zones regardless
    // of your opt-in status.
    //
    // If you do not use this parameter, the results include only the zones for
    // the Regions where you have chosen the option to opt in.
    AllAvailabilityZones *bool `type:"boolean"`
```
(we don't use the param)